ecommerce-product-page-main/styled.css

```diff
@@ -367,45 +333,29 @@ ul.list li a:hover{
 
 
 
-.button1{
-
+.discount{
     font-family: 'Kumbh Sans', sans-serif;
     font-weight: 700;
     font-size: 12px;
     color: hsl(26,100%,55%);
     background-color: hsl(26, 100%, 94%);
-    max-width: 45px;
-    max-height: 30%;
+    flex-shrink: 1;
     margin-top: 20px;
-    border-radius: 5px;
+    border-radius: 5px;  
     border-style: none;
-    
+
 }
 
 .imgbutt{

```

Comment:
How come you didn;t use margins and other to properly position your imgbutt class?

Reply:
because i already did it in a parent class